src/Sfa.Tl.ResultsAndCertification.Models/Contracts/TrainingProvider/LearnerRecordDetails.cs

```diff
@@ -15,6 +16,7 @@ public class LearnerRecordDetails
         public string ProviderName { get; set; }
         public long ProviderUkprn { get; set; }
         public string TlevelTitle { get; set; }
+        public ICollection<string> Specialism { get; set; }
```

Comment:
I think the name should be in plural => Specialisms
Why a collection? I don't think you are adding or removing anything